.github/workflows/release.yml

```diff
@@ -342,11 +342,11 @@ jobs:
           git restore packages/@biomejs/wasm-nodejs/package.json
           git restore packages/@biomejs/wasm-web/package.json
 
-      - name: Copy binaries
-        run: node packages/@biomejs/biome/scripts/copy-binaries.mjs
+      - name: Generate npm packages
+        run: node packages/@biomejs/biome/scripts/generate-packages.mjs
 
       - name: Publish npm packages as latest
-        run: for package in packages/@biomejs/*; do if [ $package != "packages/@biomejs/js-api" ]; then npm publish $package --tag latest --access public --provenance; fi; done
+        run: for package in packages/@biomejs/*; do if [ $package != "packages/@biomejs/js-api" ]; then pnpm publish $package  --no-git-checks--tag latest --access public --provenance; fi; done
```

Comment:
As pnpm doesn't support the `--provenance` flag yet, let's stick to use npm. We also stick to use `generate-packages.mjs`, which will take care of workspace protocol